content/docs/command-reference/import.md

```diff
@@ -91,13 +91,32 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-b`, `--backup` - imports files in "backup mode": see Backup Mode section for
+  details.
```

Comment:
```suggestion
- `-b`, `--backup` - turns _backup mode_ on for this import, so it can be pushed
  to remote storage (see [Backup mode](#backup-mode) for details).
```